### PR TITLE
Regenerate the GitHub personal access token for conda-forge-manager

### PR DESCRIPTION
**Pull request by conda-forge-manager**
To do this, I created the token with the following permissions (all ticked): repo, admin:org, admin:repo_hook, admin:org_hook

```

travis encrypt GH_TOKEN="..."

```

cc @pelson.

### CODE REVIEW

**Comment by pelson** [comment]
cc @jakirkham @isuruf 

Follows up from 6129

File: .travis.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ env:
...
+        # Add the GH_TOKEN environment variable. (This is the "staged-recipes" token for the conda-forge-manager user.)
+        secure: "gQ8MGFWnDxXNYKNcW9g8EaztbdmjE1n3MOIvBR9D3A3sOV+0PNb0VV7gS1NnYph+80MvWQBwZpIYC9aPxWt4uf7U1/0WZpgxrS6FRIdI85d3jwy7L3Qd94Luj1J5c0y3XAa3YhZYLx3TcMGuZYE6880vP+zkxo+jnYf5CUE6zxzttfDK/B8ps3zn+b/t0pBqrN5Xy3ZvotYGP4GSPmt2i5ZM3tnjOKTPwcBkHptq0k/nJWfrcOv8lhqea6TQDqHHtQsIqnL5pNBxMsE9mlonlMBr+fw9vImqiWl1bCzkAJ5vLi01eX6+ktpYinEx5E+SZbaRrYBIXifNONGFA87C4GgzzTG9ERHu7PRoFeZ3g9r+dd7yqggxgMtEylsJKWUOV4WtXmI97b+o/1T0+pk7luy1UgkzqXf/ArGLMeonb4gtwiBkKgbplW7sSkrZejoCkz16n2lCzLkEinCsMXOnfcmakW1cdsZJsBo9CCnA35uQuAbbb8jRBYSG3vuiwcYlXjVavrRsp7YbU5lB3Inj1bIDYNZKCbHZE5KxXiveRNwC8zGUMeetH2TW7qcb02r+ZfxdQQeTY+JwIMrbMQDgwCkFDVR2LvAJ/IrztRxEIM2GZAIalzSS5UJbWWMRxXcD4mWTXL8hJET+Tmolf/Mfk4T3Hy2+Zo7aFU08IZXa7iA="
```
Comment (jakirkham) [inline .travis.yml]: Looks like we lost the preceding `-` here. Is that expected?

**Comment by jakirkham** [comment]
Thanks @pelson. Really appreciate it. 😄

File: .travis.yml (hunk 1 above)
Comment (pelson) [reply .travis.yml]: Good spot. Have updated.

**Comment by pelson** [comment]
OK, updated this to use a travis token from the travis-ci.org *profile page*, rather than through their CLI.

**Comment by isuruf** [comment]
Thanks

**Comment by isuruf** [comment]
`github.GithubException.BadCredentialsException: 401 {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}`

**Comment by isuruf** [comment]
https://travis-ci.org/conda-forge/staged-recipes/builds/394865075

**Comment by jakirkham** [comment]
Just to be sure, those tokens are both for the same user?

**Comment by pelson** [comment]
> Just to be sure, those tokens are both for the same user?

Yep. I even updated the comments to reflect exactly which token it is...

My concern now is that I'm using an old-ish travis client. I'll update and re-run the encrypt. If they are different, that is interesting. If the same, I'm a little bit stumped. 😞

**Comment by pelson** [comment]
> If they are different, that is interesting. If the same, I'm a little bit stumped.

Woops. Forgot that they always change! 😄 
I'll re-submit the GH_TOKEN in the hope that the new client makes a difference...

**Comment by isuruf** [comment]
@pelson, did you give correct permissions to GH_TOKEN?

**Pull request merged**